chore(KFLUXVNGD-155): Add custom certificate support for git clone and buildah task (#1814)
konflux-ci/build-definitions: task-generator/trusted-artifacts/README.md

| `addResult`          | sequence of Tekton [TaskResult]s                 | Additional Tekton Task results to add to the Task |
| `addVolume`          | sequence of [Volume]s                            | Additional Volumes to add to the Task |
| `addVolumeMount`     | sequence of [VolumeMount]s                       | Additional VolumeMount to add to the Task |

> | `addTAVolumeMount`   | sequence of [VolumeMount]s                       | Additional VolumeMount to add to the Trusted Artifact Task |

Reviewer: Here as well, it would be good to say that this is specifically for the `use-trusted-artifact` step rather than the whole task

| `base`               | string                                           | Relative path from `recipe.yaml` to the Task definition of the non-Trusted Artifacts Task |
| `description`        | string                                           | Description of the Trusted Artifacts Task |
| `displaySuffix`      | string                                           | Additional text to place to the value of `tekton.dev/displayName` annotation from the non-Trusted Artifacts Task to the Trusted Artifacts Task (default: `" oci trusted artifacts"`) |
| `preferStepTemplate` | boolean                                          | When `true` preference is set to configure common configuration on the `Task.spec.stepTemplate` rather than on each Task Step |

> | `useTAVolumeMount`   | boolean                                          | When `true` Volume Mount is added to the Trusted Artifact |

Reviewer 2: We need to mention somehow that if `addTAVolumeMount` is not provided and `useTAVolumeMount` is `true` then a default mount is applied.

Author: Updated

> | `useTAVolumeMount`   | boolean                                          | When `true` and if `addTAVolumeMount` is provided then this value is added to the Trusted Artifact step, if not then default volume mount is added |

Reviewer 2: ```suggestion
| `useTAVolumeMount`   | boolean                                          | When `true` VolumeMounts are added to the Trusted Artifact step. If `addTAVolumeMount` is provided then those VolumeMounts are added. Otherwise,  default VolumeMounts are added |
```

> | `useTAVolumeMount`   | boolean                                          | When `true` VolumeMounts are added to the Trusted Artifact step. If `addTAVolumeMount` is provided then those VolumeMounts are added. Otherwise,  default VolumeMounts are added |

Reviewer 2: ```suggestion
| `useTAVolumeMount`   | boolean                                          | When `true` VolumeMounts are added to the Trusted Artifact step. If `addTAVolumeMount` is provided then those VolumeMounts are added. Otherwise, default VolumeMounts are added |
```
